### DISCUSSION TITLE
Changing Accent Color

### DISCUSSION THREAD

**Discussion by sgbx** [usememos/memos#4137] [General]
Is there a CSS snippet for easily changing accent color in light/dark theme?

**Comment by RoccoSmit** [comment]
Are you able to share more information on which colours you are wanting to change?

**Comment by sgbx** [comment]
Yes sure. I want to change the green accent color (see screenshot) to another color (including hoover color).
![Bildschirmfoto vom 2024-11-18 16-08-41](https://github.com/user-attachments/assets/9a4d571f-515a-4ba5-8739-98f14ce8996d)

**Comment by RoccoSmit** [comment]
Give this a try:
`settings` -> `system` -> `additional styles`

```
.bg-primary,
.bg-primary-darker,
.dark\:bg-primary-darker:is(.dark *) {
    background-color: red;
}

.focus-within\:ring-primary:focus-within,
.dark\:focus-within\:ring-primary-darker:focus-within:is(.dark *),
.focus-within\:border-primary:focus-within,
.dark\:focus-within\:border-primary-darker:focus-within:is(.dark *)  {
    border-color: red;
    --tw-ring-color: red;
}
```
and change the 3 references to red out with your own colour

**Reply by sgbx** [reply]
Thank you, This works (partly)
![Bildschirmfoto vom 2024-11-19 13-21-06](https://github.com/user-attachments/assets/dfdc6805-3805-481e-9343-ad3734cedac5)

**Reply by RoccoSmit** [reply]
The calendar feature got a rework recently. You'll know if you have the latest version if the calendar has month navigation selectors. 
For previous calendar versions you need to add

```
.bg-teal-500, .bg-teal-600, .bg-teal-700 {
    background-color: red;
}
```

**Reply by sgbx** [reply]
Very cool. Thank you very much, works fine!
